docs/toc.js

```diff
@@ -116,9 +116,51 @@ module.exports = {
           type: 'link',
         },
         {
-          pathSegment: 'doc-blocks',
+          pathSegment: '',
           title: 'Doc Blocks',
-          type: 'link',
+          type: 'menu',
+          children: [
+            {
+              pathSegment: 'doc-block-argstable',
+              title: 'Argstable',
```

Comment:
```suggestion
              title: 'ArgsTable',
```

Reply:
Fixed with the latest commit. Thanks for the callout, appreciate it 🙏 . Marking this as solved.